src/neptune/handler.py

```diff
@@ -546,14 +546,14 @@ def fetch(self):
         return self._pass_call_to_attr(function_name="fetch")
 
     def fetch_last(self):
-        """Fetches last value stored in the series from Neptune servers.
+        """Fetches the last value stored in the series from Neptune servers.
```

Comment:
@TytusKalicki Maybe we could also leave out `servers`, as it should be pretty clear here. Otherwise I guess we should add an article (`the Neptune servers`) but it just seems a bit verbose. What do you think?